File: sample/javaSample/src/main/java/jp/co/geniee/samples/MainActivity.java
package jp.co.geniee.samples;

import android.content.Intent;

public class MainActivity extends BaseMenuActivity {
    @Override
    protected MenuItem[] getListViewContents() {
        MenuItem[] menuItem = {
                new MenuItem("GNAd", "Banner,Native,Interstitial (deprecated),Video Ad (deprecated)", new Intent(this, jp.co.geniee.samples.gnad.MenuActivity.class)),
                new MenuItem("Fullscreen Interstitial", "", new Intent(this, jp.co.geniee.samples.fullscreeninterstitial.MainActivity.class)),
                new MenuItem("Reward Video", "", new Intent(this, jp.co.geniee.samples.rewardvideo.MainActivity.class)),
                new MenuItem("Video Player", "", new Intent(this, jp.co.geniee.samples.videoplayer.GNAdSampleVideoPlayer.class)),
                new MenuItem("Google Mediation", "Banner,Fullscreen Interstitial,Reward Video", new Intent(this, jp.co.geniee.samples.googlemediation.MenuActivity.class)),
                new MenuItem("Swipe", "In View,Ad Mob,Full Size", new Intent(this, jp.co.geniee.samples.swipe.MenuActivity.class)),
                new MenuItem("Scroll Banner", "In View,Ad Mob", new Intent(this, jp.co.geniee.samples.scrollbanner.MenuActivity.class))
        };

        return menuItem;
    }
}

File: sample/kotlinSample/src/main/java/jp/co/geniee/samples/MainActivity.kt
package jp.co.geniee.samples

import android.content.Intent

class MainActivity : BaseMenuActivity() {

    override fun getListViewContents(): Array<MenuItem> {

        return arrayOf(
            MenuItem("GNAd", "Banner,Native,Interstitial (deprecated),Video Ad (deprecated)", Intent(this, jp.co.geniee.samples.gnad.MenuActivity::class.java)),
            MenuItem("Firebase", "Banner,Interstitial,Reward", Intent(this, jp.co.geniee.samples.firebasegooglead.MainActivity::class.java))
        )
    }
}
